onboarding/src/Components/utils.js

```diff
@@ -64,8 +79,10 @@ export function tryFetchJson(res, noJsonMessage){
 	if(typeof noJsonMessage === 'string')
 		noJsonObj.detail = noJsonMessage;
 
-	if(res.ok){
-		result = res.json().catch(() => {return noJsonObj});
+	if(!res.ok){
```

Comment:
Why negate the logic of `if`?

Reply:
Maybe I didn't understand this code - for me it looked like if there is error, catch it and return error message, else return res.text()? I think now I see what you meant, so I will have some lines of code to change...

Reply:
The logic was:
if it's ok - `if res.ok` then try `json()` otherwise take the `text()`.
But when you try `json()` be careful if it really can be 'jsonified' ;-)